app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -76,7 +76,7 @@
     RecyclerView rvChatFeed;
 
     @BindView(R.id.et_message)
-    EditText etMessage;
+    EditText ChatMessage;
```

Comment:
variable names should start with a lowercase letter, so please follow the codestyle now onwards.